src/locales/pt.json

```diff
@@ -0,0 +1,217 @@
+{
+  "Blockchain": {
+    "Mnemonic": "Mnemônico",
+    "Address": "Endereço",
+    "Wallet": "Carteira",
+    "Wallets": "Carteiras",
+    "Coins": "Moedas",
+    "Tokens": "Tokens",
+    "Balance": "Saldo",
+    "Tx hash": "Hash Tx",
+    "Gas": "Gas",
+    "Tax": "Imposto",
+    "Fee": "Taxa",
+    "Memo": "Memo",
+    "decimals": "decimais",
+    "Buy": "Comprar",
+    "Exchanges": "Exchanges",
+    "Fiat": "Fiat",
+    "APY": "APY"
+  },
+  "Form": {
+    "Amount": "Quantidade",
+    "Recipient": "Destinatário",
+    "Name": "Nome",
+    "optional": "opcional",
+    "Submit": "Enviar",
+    "Confirm": "Confirmar",
+    "Reset": "Reset"
```

Comment:
We do use both, only a suggestion for `Apagar`, instead. :)

Reply:
But "Apagar" is more in the direction of erase, and reset has an effect to undo, or rollback to default.